RecoTracker/TkDetLayers/src/TkDetUtil.cc

```diff
@@ -40,7 +49,7 @@ namespace tkDetUtil {
       auto y1 = hori ? yc - maxDistance.y() :  xc - maxDistance.x();
       auto x1 = hori ? xc + maxDistance.x() : -yc + maxDistance.y();
 
-      dphi = std::acos( (x0*x1+y0*y1)/std::sqrt((x0*x0+y0*y0)*(x1*x1+y1*y1)) );
+      dphi = clamped_acos((x0*x1+y0*y1)/std::sqrt((x0*x0+y0*y0)*(x1*x1+y1*y1)));
```

Comment:
Only "offtrack" or "ontrack" as well?
OffTrack should be defined as not on track so we should see a difference in the number of hits on track (that does not seems to be the case)
I do not see how "computeWindowSize" can affect DQM w/o affecting tracks as well